test/test_helper.rb

```diff
@@ -2,46 +2,54 @@
 
 ENV["RAILS_ENV"] = "test"
 
+require 'warning'
+Warning.ignore(/.*/, %r{.*lib/shoulda/matchers/active_model/allow_value_matcher/attribute_setter.rb})
+Warning.ignore(/.*/, %r{.*lib/shoulda/context/context.rb})
+Warning.ignore(/.*/, %r{.*lib/mail/parsers.*})
+
 require File.expand_path('dummy/config/environment.rb', __dir__)
+
 require "rails/test_help"
 require "rr"
 require "shoulda"
 require "minitest/unit"
 require 'minitest/reporters'
 
-junit_output_dir = ENV['JUNIT_OUTPUT'].presence || "test/reports"
-Minitest::Reporters.use! [
-  Minitest::Reporters::DefaultReporter.new,
-  Minitest::Reporters::JUnitReporter.new(junit_output_dir)
-]
+unless ENV['RM_INFO']
```

Comment:
fyi - to support running minitest within RubyMine IDE. More info at https://youtrack.jetbrains.com/issue/RUBY-28083